src/agentscope/service/web/tripadvisor.py

```diff
@@ -0,0 +1,526 @@
+# -*- coding: utf-8 -*-
+"""TripAdvisor APIs for searching and retrieving location information."""
+
+from loguru import logger
+import requests
+from agentscope.service.service_response import ServiceResponse
+from agentscope.service.service_status import ServiceExecStatus
+
+
+def get_tripadvisor_location_photos(
+    api_key: str,
+    location_id: str = None,
```

Comment:
What's a valid location id and how to obtain it? Can I use the location name here, e.g. Beijing?

Reply:
Btw, change its typing to int if it only receives integer to avoid misunderstanding

Reply:
Q2: yes `location_id` is an integer, but since `tripadvisor_search` returns `location_id` as a string, here I also use string type to standardize things.